scripts/qual_validation/qual_validation.py

```diff
@@ -59,19 +63,20 @@
     gpu_profile_dir = gpu_profile
 else:
     gpu_profile_dir = f"{output}/gpu_profile"
-    subprocess.run(f"spark_rapids_user_tools onprem profiling --csv {jar_arg} --local_folder {gpu_profile_dir} --eventlogs {gpu_log}", shell=True)
+    subprocess.run(f"spark_rapids profiling --csv {jar_arg} --output_folder {gpu_profile_dir} --eventlogs {gpu_log} {verbose}", shell=True)
 
 ### run CPU profiler if needed
 cpu_profile_dir = ""
 if cpu_profile is not None:
     cpu_profile_dir = cpu_profile
 else:
     cpu_profile_dir = f"{output}/cpu_profile"
-    subprocess.run(f"spark_rapids_user_tools onprem profiling --csv {jar_arg} --local_folder {cpu_profile_dir} --eventlogs {cpu_log}", shell=True)
+    subprocess.run(f"spark_rapids profiling --csv {jar_arg} --output_folder {cpu_profile_dir} --eventlogs {cpu_log} {verbose}", shell=True)
 
 ### run CPU qualification with xgboost model
 cpu_tmp_dir = f"{output}/cpu"
-subprocess.run(f"spark_rapids_user_tools onprem qualification {jar_arg} --estimation_model xgboost --local_folder {cpu_tmp_dir} --eventlogs {cpu_log}", shell=True)
+print(f"spark_rapids qualification --platform {platform} {jar_arg} --estimation_model xgboost --output_folder {cpu_tmp_dir} --eventlogs {cpu_log} {verbose}")
+subprocess.run(f"spark_rapids qualification --platform {platform} {jar_arg} --estimation_model xgboost --output_folder {cpu_tmp_dir} --eventlogs {cpu_log} {verbose}", shell=True)
```

Comment:
minor nit: we can store command `spark_rapids qualification --platform {platform} {jar_arg} --estimation_model xgboost --output_folder {cpu_tmp_dir} --eventlogs {cpu_log} {verbose}` into a variable and reuse here?

Reply:
Good catch.  I'll just remove that extra print statement as it was only used for debugging.